src/smartResources/utils.ts

```diff
@@ -0,0 +1,220 @@
+import { Extension, Element, Coding, CodeableConcept, code, HumanName, HumanNameNameUse, Period } from "../definition";
+
+const TRANSLATION_URL = 'http://hl7.org/fhir/StructureDefinition/translation';
+const LANG_URL = 'lang';
+const CONTENT_URL = 'content';
+
+const REG_EXP_UUID = new RegExp(/^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i);
```

Comment:
Perhaps it would be better if 
const REG_EXP_UUID = new RegExp(/^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/i); is used, because uuid are case insensitive.